app/views/shared/_header.html.haml

```diff
@@ -6,14 +6,14 @@
       %li= link_to "Videos"
       %li= link_to "My Queue"
       %li= link_to "People"
-    =form_tag(search_videos_path, class: "col-md-5 navbar-form form-group" ) do
+    =form_tag(search_videos_path,method:'GET', class: "col-md-5 navbar-form form-group" ) do
```

Comment:
Inconsistent style here. missing spaces before `form_tag` and `method` while there is 1 more space after class.